sdk/typescript/src/multisig/publickey.ts

```diff
@@ -0,0 +1,289 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { fromB64, toB64 } from '@mysten/bcs';
+import { blake2b } from '@noble/hashes/blake2b';
+import { bytesToHex } from '@noble/hashes/utils';
+import { PublicKey, bytesEqual } from '../cryptography/publickey.js';
+import type {
+	SerializedSignature,
+	SignatureFlag,
+	SignatureScheme,
+} from '../cryptography/signature.js';
+import {
+	SIGNATURE_FLAG_TO_SCHEME,
+	SIGNATURE_SCHEME_TO_FLAG,
+	parseSerializedSignature,
+} from '../cryptography/signature.js';
+import { normalizeSuiAddress } from '../utils/sui-types.js';
+import { builder } from '../builder/bcs.js';
```

Comment:
Same BCS question here

Reply:
same answer here, we need to definitions for the multisig structs